### PR TITLE
Stub for supporting operator CSV modifications

### PR DESCRIPTION
**Pull request by MartinBasti**
Preparation for future implementation. This commit only connects
osbs-client and atomic-reactor changes.

* CLOUDBLD-3976

Signed-off-by: Martin Bašti <mbasti@redhat.com>

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered
- [n/a] JSON/YAML configuration changes are updated in the relevant schema
- [n/a] Changes to metadata also update the documentation for the metadata
- [n/a] Pull request has a link to an osbs-docs PR for user documentation updates
- [n/a] New feature can be disabled from a configuration file

### CODE REVIEW

**Comment by tkdchen** [comment]
@MartinBasti What's the next step for this PR? Are you going to merge it?

File: atomic_reactor/plugins/pre_check_user_settings.py [hunk 1]
```diff
@@ -101,9 +102,26 @@ def validate_user_config_files(self):
         read_fetch_artifacts_url(self.workflow)
         read_content_sets(self.workflow)
 
+    def isolated_from_scratch_build(self):
+        """Isolated builds for FROM scratch builds are prohibited
+         except operator bundle images"""
+        if (
```
Comment (rcerven) [inline atomic_reactor/plugins/pre_check_user_settings.py:108]: and where is check for FROM scratch??

Comment (MartinBasti) [reply atomic_reactor/plugins/pre_check_user_settings.py:108]: It stayed at original place

Comment (MartinBasti) [reply atomic_reactor/plugins/pre_check_user_settings.py:108]: got it now

**Review by rcerven — APPROVED** [review]

**Comment by mkosiarc** [comment]
do you know why the coverage dropped only for centos ?

**Comment by MartinBasti** [comment]
> do you know why the coverage dropped only for centos ?

I dunno

**Review by mkosiarc — APPROVED** [review]

**Comment by MartinBasti** [comment]
RN (valid when feature is finished):
- Allow modification in Operator CSV files for isolated builds

**Pull request merged**